### DISCUSSION TITLE
Installed, but I don't know, how to SEE it in Thunderbird

### DISCUSSION THREAD

**Discussion by feuerholt** [protz/thunderbird-conversations#2081] [Q&A] [answered]
I have installed and configured. But then...?
1. Where can I see the new view/conversation?
2. In which menu items and context menus is it integrated?
3. Which view etc. do I need to have set so that I can see and use the plugin?

Thanks
Thunderbird 115.8 (64 bit), Win 10

**Comment by Standard8** [comment, marked as answer]
When you click on a message in the message list, then you'll see the message and any related messages in the preview window - see the images on https://addons.thunderbird.net/en-GB/thunderbird/addon/gmail-conversation-view/?src=hp-dl-featured for example.

You do not need to enable anything separately once it is installed.
